bear/main.py.in

```diff
@@ -262,7 +262,11 @@ def parse_exec_trace(filename):
 
     logging.debug('parse exec trace file: %s', filename)
     with open(filename, 'r') as handler:
-        entry = json.load(handler)
+        try:
+            entry = json.load(handler)
+        except ValueError:
+            logging.debug('unable to parse: %s', filename)
+            return Execution(pid='', cwd='', cmd='')
```

Comment:
yep, that was it.  So looks like the problematic string was:

```
{ "pid": 14328, "cmd": [ "scripts/extract-cert"
```

which looks like some kind of cut off thing. So we probably don't want this patch yet, though I'm getting closer.